### ISSUE TITLE
enableDiskCache as a config parameter

### ISSUE THREAD

**Issue by ghost** [mttkay/droid-fu#23]
imageCache.enableDiskCache(context, ImageCache.DISK_CACHE_SDCARD);

is hardcoded in ImageLoader. Could enableDiskCache be a config parameter?

**Comment by bumbobway** [comment]
I agree, the param should be for either SD Card or application data.
